New package: SIRIUS v0.1.0

JuliaRegistrator opened the pull request:
- Registering package: SIRIUS
- Repository: https://github.com/abussy/SIRIUS.jl
- Created by: @abussy
- Version: v0.1.0
- Commit: 83e17086978b30ff62005be254574f454efeac73
- Reviewed by: @abussy
- Reference: https://github.com/abussy/SIRIUS.jl/commit/83e17086978b30ff62005be254574f454efeac73#commitcomment-147954302
- Description: Julia wrappers for the SIRIUS library.
<!-- bf0c69308befbd3ccf2cc956ac8a46712550b79fc9bfb5e4edf8f833f05f4c18b06eddad8845b45beb9f45c2b8020dd6f8653167582184f615f63e31ee2486b90cffed71cf7ddaa484659d4249e02fcff63e6560d46c0524f3a6959a3dc29913c46afeba44e245e900225af7123fa7c87404154ab7f65796367d635e15ad9fe5936ea4663fa5be2fec8e29375bb91a1669b6d3f4bc88e81e4b8b3d321dcd4ec64e306e019c444fc52beabf9cb734d856e7773f0163522598ae3b70bc82ffa436bc7fe4e4aa4a7dfb5274479476bf07df812210835057a407bd3b2ad318a7ff36 -->

goerz commented:
Should be ok, but you will have to ask on Slack for a manual merge after the end of the three-day-period

giordano commented:
> The following dependencies do not have a [compat] entry that is upper-bounded and only includes a finite number of breaking releases: JSON3, MPI, OpenBLAS32

JuliaRegistrator committed:
New package: SIRIUS v0.1.0

UUID: 3581e6f0-5a2a-4fbf-9499-336905df69aa
Repo: https://github.com/abussy/SIRIUS.jl.git
Tree: 0e1b2cac89f5bd63f36ce3f298686f04d12c9015

Registrator tree SHA: 17aec322677d9b81cdd6b9b9236b09a3f1374c6a
